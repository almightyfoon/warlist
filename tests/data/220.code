c4501b0x19191h1he2c37y9u1Q1Tgege___-0B0Z19193ci2dWdWhZh_hZh_bB