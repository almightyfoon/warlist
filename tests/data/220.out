https://conflictchamber.com/#c4501bkv19191h1he2c37y9u1Q1Tkx___-0B0Z19193ci2dWdWhZh_hZh_bB

Cryx Army - 70 / 75 points

[Deneghra 1] Warwitch Deneghra [+26]
 - Deathripper [6]
 - Deathripper [6]
 - Stalker [8]
 - Stalker [8]
Aiakos, Scourge of the Meredius [4]
 - Kraken [36]
Orin Midwinter, Rogue Inquisitor [5]
Ragman [4]
Satyxis Blood Witches (min) [8]
 - Satyxis Blood Hag [4]
Satyxis Gunslingers [7]

Cryx Army - 75 / 75 points
[Theme] Dark Host

[Coven 1] The Witch Coven of Garlghast [+26]
 - Deathripper [6]
 - Deathripper [6]
 - Desecrator [14]
Bane Lord Tartarus [0(6)]
Bane Riders (max) [20]
Bane Riders (max) [20]
Bane Warriors (min) [10]
 - Bane Warrior Officer & Standard [0(5)]
Bane Warriors (min) [10]
 - Bane Warrior Officer & Standard [0(5)]
Wraith Engine [15]
